### PR TITLE
add missing link to planets-data.txt file

### PR DESCRIPTION
**Pull request by flora-le**
Link was missing, add a link to download the file planets-data.txt

### Description

Link was missing for downloading the file planets-data.txt, add a link to download the file.

### Motivation

To access the link to download the file when clicking on the name

### CODE REVIEW

File: files/fr/learn/html/tables/structuring_planet_data/index.md [hunk 1]
```diff
@@ -46,7 +46,7 @@ Vous pouvez aussi [regarder l'exemple ici](https://mdn.github.io/learning-area/h
 
 ## Étapes à suivre
 
-Les étapes suivantes décrivent ce que vous devez faire pour compléter l'exemple de tableau. Toutes les données dont vous avez besoin sont contenues dans le fichier `planets-data.txt`. Si vous avez du mal à visualiser les données, regardez l'exemple donné dans le lien ci-dessus, ou essayez de dessiner un diagramme.
+Les étapes suivantes décrivent ce que vous devez faire pour compléter l'exemple de tableau. Toutes les données dont vous avez besoin sont contenues dans le fichier [planets-data.txt](https://github.com/mdn/learning-area/blob/main/html/tables/assessment-start/planets-data.txt). Si vous avez du mal à visualiser les données, regardez l'exemple donné dans le lien ci-dessus, ou essayez de dessiner un diagramme.
```
Comment (cw118) [inline files/fr/learn/html/tables/structuring_planet_data/index.md:49]:
```suggestion
Les étapes suivantes décrivent ce que vous devez faire pour compléter l'exemple de tableau. Toutes les données dont vous avez besoin sont contenues dans le fichier [`planets-data.txt`](https://github.com/mdn/learning-area/blob/main/html/tables/assessment-start/planets-data.txt). Si vous avez du mal à visualiser les données, regardez l'exemple donné dans le lien ci-dessus, ou essayez de dessiner un diagramme.
```

(keep the `code` formatting)

**Review by cw118 — CHANGES_REQUESTED** [review]
Thank you again for this PR, I added one small suggestion :)

**Comment by cw118** [comment]
@flora-le Please don't hesitate if help is needed on this PR :)

**Comment by flora-le** [comment]
Hello,
thanks! it's my first time doign this kind of thing. Does clicking on the button "update branch" is enough ? Thanks :)

**Review by SphinxKnight — APPROVED** [review]
Merci @flora-le :)

**Pull request merged**